fetch.bs

```diff
@@ -4303,14 +4332,19 @@ Range Requests</cite>. [[HTTP-RANGE]] However, this is not widely supported by b
     <var>httpRequest</var>'s
     <a for=request>credentials mode</a>.
 
+   <li>
+    <p>Let <var>httpCache</var> be the result of <a>determining the HTTP cache partition</a> for
+    <var>httpRequest</var>. If <var>httpCache</var> is null, fetch behaves as if there is no HTTP cache
+    at all and the <a>cache mode</a> is considered "<code>no-cache</code>".
+
    <li>
     <p>If <var>httpRequest</var>'s <a for=request>cache mode</a> is neither "<code>no-store</code>"
     nor "<code>reload</code>", then:
 
     <ol>
      <li>
-      <p>Set <var>storedResponse</var> to the result of selecting a response from the HTTP cache,
-      possibly needing validation, as per the
+      <p>Set <var>storedResponse</var> to the result of selecting a response from the
+      <var>httpCache</var>, possibly needing validation, as per the
```

Comment:
This step will be reached even if the mode is no-cache. Is that desired? Maybe no-cache is not correct, and it should be no-store instead?

Your goal is to prevent any steps that reference _httpCache_ from being reached if _httpCache_ is null, because that would cause a null pointer exception.

Reply:
Ah that's right, it should be no-store. Updated.